manifests/pulp.pp

```diff
@@ -129,50 +134,79 @@
     }
   }
 
+  if $expose_mongodb {
```

Comment:
```suggestion
  if $manage_mongodb and $expose_mongodb {
```

Reply:
Expose mongodb should fail if manage_mongodb is false... should I go the explicit fail route instead?

Reply:
I think that makes sense.